tests/common/test_json.py

```diff
@@ -226,6 +226,17 @@ def test_json_pendulum(json_impl: SupportsJson) -> None:
 #     print(str(delta.as_timedelta()))
 
 
+@pytest.fixture(autouse=True)
+def run_around_tests():
+    for impl in _JSON_IMPL:
+        # clear the encoder
+        impl.set_custom_encoder(None)
+    yield
```

Comment:
you must put `yield` in try..finally... and release changes there. you may get exception form the code executed on yield

Reply:
changed in 659cc5c